### DISCUSSION TITLE
modu:lenotfounderror: no module named 'kombu.transport.django'

### DISCUSSION THREAD

**Discussion by Mahesh935525** [celery/kombu#1837] [Q&A] [answered]
Hi, 

I want to upgrade the old kombu to the new kumbo version, currently, I'm using kombu<3.1,>=3.0.34 in the project and upgraded to the new version kombu==5.3.4
after changing the version I'm getting the below error, please suggest on this error. 

**Error:** 

**no module named 'kombu.transport.django'**

The **INSTALLED_APPS** from **seetings.py** file

INSTALLED_APPS = (
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.sites',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    # Uncomment the next line to enable the admin:
    'django.contrib.admin',
    # Uncomment the next line to enable admin documentation:
    # 'django.contrib.admindocs',
    'gunicorn',
    'kombu.transport.django',
    #'django_celery_results',
    'djcelery',
    'jobsqueue'
)

**Old version packages ->**

python=2.7
amqp<2.0,>=1.4.9
celery==3.1.23
dj-database-url==0.2.1
django-celery==3.0.11
Django==1.4.3
kombu<3.1,>=3.0.34

**New version ->**

python==3.10.0
amqp==5.2.0
celery==5.3.6
billiard==3.6.3.0
anyjson==0.3.3
dj-database-url==0.2.1
django-celery==3.3.1
Django==5.0
gdata==2.0.17
geopy==2.4.1
gspread==0.3.0
gunicorn==21.2.0
imageio==2.33.1
kombu==5.3.4
vine==5.1.0

**Comment by auvipy** [comment, marked as answer]
we do not support django transport for more then 8 years now
